src/index.ts

```diff
@@ -392,6 +395,76 @@ export function apply(ctx: Context, config: Config) {
               r2: true,
             }
           }
+          case 'comfyui': {
+            const workflowText2Image = config.workflowText2Image ? resolve(ctx.baseDir, config.workflowText2Image) : '../data/default-comfyui-t2i-wf.json'
+            const workflowImage2Image = config.workflowImage2Image ? resolve(ctx.baseDir, config.workflowImage2Image) : '../data/default-comfyui-i2i-wf.json'
+            const workflow = image ? workflowImage2Image : workflowText2Image
+            logger.debug('workflow:', workflow)
+            const prompt= require(workflow)
```

Comment:
你都改 async 了这里不要 require 比较好。

```suggestion
            const prompt = JSON.parse(await readFile(workflow, 'utf8'))
```

Reply:
commit 66bdd52441685105d971ea0975f15ee3ec699a49
为readFile修改了一下路径的获取